python/lsst/ctrl/bps/htcondor/htcondor_service.py

```diff
@@ -959,11 +959,13 @@ def _replace_cmd_vars(arguments, gwjob):
     """
     try:
         arguments = arguments.format(**gwjob.cmdvals)
-    except (KeyError, TypeError):  # TypeError in case None instead of {}
-        _LOG.error(
-            "Could not replace command variables:\narguments: %s\ncmdvals: %s", arguments, gwjob.cmdvals
-        )
+    except KeyError:
+        _LOG.error("Could not replace command variables")
         raise
+    except TypeError:
+        _LOG.debug("Command variables not provided; assuming there are no command variables to replace")
```

Comment:
"assuming"?  Couldn't this be checked?  Would it be better to pass an empty dictionary to the format function instead of None which would actually raise a KeyError if there are any command variables to replace.